ci-operator/step-registry/openshift-logging/enable-cluster-logging/openshift-logging-enable-cluster-logging-commands.sh

```diff
@@ -0,0 +1,582 @@
+#!/bin/bash
+set -o nounset
+
+storage_class=""
+minio_secret=""
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+# setup proxy
+if test -f "${SHARED_DIR}/proxy-conf.sh"
+then
+    source "${SHARED_DIR}/proxy-conf.sh"
+fi
+
+
+### common function ###
+function prepare_namespace(){
+    echo "Prepare namespace $1"
+    if oc get project $1 -o name; then
+        echo "use existing namespace $1"
+    else
+        echo "create namespace $1"
+        oc adm new-project $1  || exit 1
+    fi
+    oc label ns $1 openshift.io/cluster-monitoring="true"
+    oc label ns $1 openshift.io/log-alerting="true"
+}
+
+function are_operators_ready(){
+    echo " Wait the operators ready up to 1 minutes"
+    clo_ready=false
+    lo_ready=false
+    count=1
+    while [[ $count -lt 4 ]]; do
+        if oc -n openshift-logging wait pod --for=condition=Ready  -l name=cluster-logging-operator; then
+            clo_ready=true
+        fi
+        if oc -n openshift-operators-redhat wait pod --for=condition=Ready  -l name=loki-operator-controller-manager; then
+            lo_ready=true
+        fi
+        if [[ $clo_ready == "true" && $lo_ready == "true" ]]; then
+            break
+        fi
+        sleep 15s
+        (( count=count + 1 ))
+    done
+    
+    if [[ $clo_ready == "false" ]]; then
+       echo "show status in openshift-logging "
+       oc -n openshift-logging get sub
```

Comment:
```suggestion
       oc -n openshift-logging get sub -oyaml
```
The yaml file may contain more info.

Reply:
drop the code. Only check if operator is deployed or not